build.gradle

```diff
@@ -34,7 +34,7 @@ repositories {
 }
 
 dependencies {
-    api 'com.blackduck.integration:blackduck-common-api:2023.4.2.9'
+    api 'com.blackduck.integration:blackduck-common-api:2023.4.2.12'
     api 'com.blackduck.integration:phone-home-client:7.0.0'
```

Comment:
For anyone who is wondering, 2023.4.2.10 was a change I made to the date format, and 2023.4.2.11 was reverting that change to the date format.